ext/zlib/zlib.c

```diff
@@ -1320,8 +1357,33 @@ static PHP_MINIT_FUNCTION(zlib)
 	php_output_handler_conflict_register(ZEND_STRL("ob_gzhandler"), php_zlib_output_conflict_check);
 	php_output_handler_conflict_register(ZEND_STRL(PHP_ZLIB_OUTPUT_HANDLER_NAME), php_zlib_output_conflict_check);
 
-	le_deflate = zend_register_list_destructors_ex(deflate_rsrc_dtor, NULL, "zlib.deflate", module_number);
-	le_inflate = zend_register_list_destructors_ex(inflate_rsrc_dtor, NULL, "zlib.inflate", module_number);
+	zend_class_entry inflate_ce;
+	INIT_CLASS_ENTRY(inflate_ce, "InflateContext", class_InflateContext_methods);
```

Comment:
I'm wondering if `InflateContext ` is really a good naming choice: if we later want to add an OO API, then `InflateContext::add` seems a bit weird for me. Maybe we should prefer `Inflator`/`Deflator`?

Reply:
The current name sound ok to me.